eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -1,20 +1,22 @@
 FROM buildpack-deps:jessie-curl
 
 #
-# Install apt packages needed by backbeat and buildbot_worker
+# Install apt packages needed by utapi and buildbot_worker
 #
 ENV LANG C.UTF-8
 COPY utapi_packages.list buildbot_worker_packages.list /tmp/
 
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+RUN wget https://nodejs.org/dist/v10.22.0/node-v10.22.0-linux-x64.tar.gz \
```

Comment:
What about using an env value for the node version? Feels like something worth parameterizing.